R/dist_spec.R

```diff
@@ -952,7 +991,8 @@ is_constrained.multi_dist_spec <- function(x, ...) {
 #' @examples
 #' LogNormal(mean = 4, sd = 1)
 #' LogNormal(mean = 4, sd = 1, max = 10)
-#' LogNormal(mean = Normal(4, 1), sd = 1, max = 10)
+#' # If specifying uncertain parameters, use the natural parameters
+#' LogNormal(meanlog = Normal(4, 1), sdlog = 1, max = 10)
```

Comment:
Perhaps I'm being paranoid here but could this lead to confusion with people thinking meanlog and sdlog corresponds to the mean and sd in the other examples? I'd suggest to use some other values here just to be sure.

```suggestion
#' LogNormal(meanlog = Normal(1.5, 0.5), sdlog = 0.25, max = 10)
```

Reply:
You're not. I thought of it too but glad you picked it up.